### PR TITLE
Handler test for MsgCreate/Edit DataSource

### LINKED ISSUE #1696: Handler test for MsgCreate/Edit DataSource
After @Benzbeeb implement new way to extract and store data as file, we need to make sure our code has test on handler flow

### PR DESCRIPTION
**Pull request by Benzbeeb**
Fixed #1696

### CODE REVIEW

File: chain/x/oracle/handler_test.go [hunk 1]
```diff
@@ -33,59 +31,155 @@ import (
...
+	zw.Write(executable)
+	zw.Close()
+	wrongGzippedExecutable := buf.Bytes()[:5]
+	msgEdit = types.NewMsgEditDataSource(1, Owner.Address, newName, newDescription, wrongGzippedExecutable, Owner.Address)
+	res, err = oracle.NewHandler(keeper)(ctx, msgEdit)
+	require.Error(t, err)
+	require.Nil(t, res)
+}
 
 // func TestEditDataSourceByNotOwner(t *testing.T) {
```
Comment (sorawit) [inline chain/x/oracle/handler_test.go]: Let's remove this code

File: chain/x/oracle/handler_test.go [hunk 2]
```diff
@@ -1,26 +1,24 @@
...
-// 	owner := sdk.AccAddress([]byte("owner"))
-// 	name := "data_source_1"
-// 	description := "description"
-// 	executable := []byte("executable")
-// 	sender := sdk.AccAddress([]byte("sender"))
-// 	msg := types.NewMsgCreateDataSource(owner, name, description, executable, sender)
-// 	return handleMsgCreateDataSource(ctx, keeper, msg)
-// }
-
 // func mockOracleScript(ctx sdk.Context, keeper Keeper) (*sdk.Result, error) {
```
Comment (sorawit) [inline chain/x/oracle/handler_test.go:18]: Let's remove it not used

**Review by sorawit — CHANGES_REQUESTED** [review]

**Review by sorawit — DISMISSED** [review]

File: chain/x/oracle/handler_test.go (hunk 2 above)
Comment (Benzbeeb) [reply chain/x/oracle/handler_test.go:18]: Will remove in #1712

File: chain/x/oracle/handler_test.go [hunk 3]
```diff
@@ -33,94 +31,155 @@ import (
...
-// 	require.Equal(t, []byte("executable"), dataSource.Executable)
-
-// 	events := ctx.EventManager().Events()
-// 	require.Equal(t, 1, len(events))
-// 	require.Equal(t, sdk.Event{
-// 		Type:       EventTypeCreateDataSource,
-// 		Attributes: []tmkv.Pair{{Key: []byte(AttributeKeyID), Value: []byte("1")}},
-// 	}, events[0])
-// }
+func deleteFile(data []byte) {
```
Comment (taobun) [inline chain/x/oracle/handler_test.go]: Move to common_test

**Review by taobun — CHANGES_REQUESTED** [review]

**Pull request merged**